lib/ui/views/schedule_view.dart

```diff
@@ -102,6 +102,15 @@ class _ScheduleViewState extends State<ScheduleView>
                               : _buildEventList(model.selectedDateEvents))
                     ],
                   ),
+                  Positioned(
+                    bottom: 16.0,
+                    right: 16.0,
+                    child: FloatingActionButton(
+                      onPressed: model.refresh,
+                      backgroundColor: AppTheme.etsLightRed,
+                      child: const Icon(Icons.refresh),
+                    ),
+                  )
```

Comment:
Instead of adding it like this, we should modify `BaseScaffold` to accept a FAB (and a FAB position) because `Scaffold` already support (and position) a FAB perfectly. What do you think @AlexandreMarkus ?

Reply:
Yes, I will do this instead

Reply:
done!